index.js

```diff
@@ -82,6 +82,7 @@ export function execa(file, args, options) {
 	logCommand(escapedCommand, parsed.options);
 
 	validateTimeout(parsed.options);
+	validateInputOptions(parsed.options);
```

Comment:
That's the main change: `input`/`inputFile` option validation now happens before process spawning, not after.